Selenium/exercise11.rb

```diff
@@ -0,0 +1,15 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path="C:/Ruby32-x64/Drivers/chromedriver.exe"
+driver = Selenium::WebDriver.for :chrome
+
+driver.get 'https://the-internet.herokuapp.com/tables'
+
+table = driver.find_element(:id,'table1')
+t_row = table.find_elements(:tag_name,'tr')
```

Comment:
This variable should be named `t_rows` instead of `t_row` as it stores an array of row elements